src/actions/transferActions.js

```diff
@@ -348,9 +409,10 @@ function cancelTransferTransactionHashRetrieved (txRequest) {
 
 function submitTx (txRequest) {
   return (dispatch, getState) => {
+    const accountInfo = getState().walletReducer.wallet.ledger.crypto[txRequest.cryptoType][0]
```

Comment:
what if wallet.ledger.crypto = null ?

Reply:
should this step depend on txRequest's cryptoType?

Reply:
suggestion: 

```js
txRequest.accountInfo = txRequest.cryptoType === 'bitcoin' ? getState().walletReducer.wallet.ledger.crypto[txRequest.cryptoType][0]
```

Reply:
wallet.ledger.crypto will not be null, it's initialized to be {}

Reply:
I dont think putting the addressPool (aka addresses that have utxos) inside the txRequest is reaonable. It's not part of the txRequest, but rather side infomation to complete txRequest.